Patches/PlayerControlPatch.cs

```diff
@@ -2504,6 +2507,8 @@ public static void Postfix(PlayerControl __instance)
             LocateArrow.OnFixedUpdate(player);
         }
 
+        if (Kamikaze.IsEnable)
+            Kamikaze.MurderKamikazedPlayers(player);
```

Comment:
As this isn't a priority, put this into`if (!lowLoad)` and this should only run on the host's side, so move it into `if (AmongUsClient.Instance.AmHost)`